script/DeployAllSandbox.s.sol

```diff
@@ -167,4 +186,11 @@ contract DeployAllSandbox is Script {
 
         vm.stopBroadcast();
     }
+
+    function deployWithCreate2(bytes32 salt, bytes memory bytecode) internal returns (address addr) {
```

Comment:
Is there a reason to do this with a custom method rather than `new Contract{ salt: 0x }()`?

Reply:
Not really. I updated to the solidity in-built